### ISSUE TITLE
Continue training model

### ISSUE THREAD

**Issue by yh675** [Raj-08/tensorflow-object-contour-detection#23]
Hi, Thanks for your work!

Is there anyway to continue training a previous model instead of starting over again?

I tried to set the tf_initial_checkpoint to my previously trained model but it doesn't seem to use my previous model given the loss functions.

**Comment by Raj-08** [comment]
Hi , currently i have not supported any checkpoints but it is in my todos. Keep a watch , it would be up soon , thank you!
